lib/Dialect/Seq/SeqOps.cpp

```diff
@@ -207,60 +208,75 @@ ParseResult CompRegOp::parse(OpAsmParser &parser, OperationState &result) {
       return failure();
   }
 
-  SmallVector<OpAsmParser::UnresolvedOperand, 4> operands;
+  constexpr size_t clockEnabledOffset = (size_t)ClockEnabled;
```

Comment:
nit: I'd prefer `constexpr size_t ceOperandOffset= ClockEnabled ? 1 : 0;` just to be explicit and more readable.